### PR TITLE
Add linux utils

### PR DESCRIPTION
**Pull request by vsafonkin**
# Description
Add a few linux utilities.

Size: ~ 3 Mb
Time: ~ 1 min

#### Related issue:
#986 

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Comment by al-cheb** [comment]
Can we put all utils in array to update them only at one place (currently we have two )?

File: images/linux/scripts/installers/basic.sh [hunk 1]
```diff
@@ -150,6 +150,18 @@ apt-get install -y --no-install-recommends gnupg2
...
+
+echo "Install flex"
+apt-get install -y --no-install-recommends flex
+
+echo "Install texinfo"
+apt-get install -y --no-install-recommends texinfo
+
 # Run tests to determine that the software installed as expected
 echo "Testing to make sure that script performed as expected, and basic scenarios work"
 for cmd in curl file ftp jq netcat ssh parallel rsync shellcheck sudo telnet time unzip wget zip; do
```
Comment (al-cheb) [inline images/linux/scripts/installers/basic.sh:167]: Please add these utils to the test loop

Comment (vsafonkin) [reply images/linux/scripts/installers/basic.sh]: Done

File: images/linux/scripts/installers/basic.sh [hunk 2]
```diff
@@ -45,114 +77,9 @@ fi
...
-echo "Install gnupg2"
-apt-get install -y --no-install-recommends gnupg2
-
-echo "Install lib32z1"
-apt-get install -y --no-install-recommends lib32z1
-
 # Run tests to determine that the software installed as expected
 echo "Testing to make sure that script performed as expected, and basic scenarios work"
-for cmd in curl file ftp jq netcat ssh parallel rsync shellcheck sudo telnet time unzip wget zip; do
+for cmd in curl file ftp jq netcat ssh parallel rsync shellcheck sudo telnet time unzip wget zip m4 bison flex texinfo; do
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/basic.sh:82]: Is it possible to use to use array `common_packages` here?
And at the bottom with `DocumentInstalledItemIndent`.

Comment (vsafonkin) [reply images/linux/scripts/installers/basic.sh]: Done.

**Pull request merged**